ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -1931,6 +1936,9 @@ class EclCmdZapGen : public EclCmdCommon
             if (!optPassword.isEmpty())
                 curlCommand.append(":").append(optPassword.get());
         }
+        if (optCreateDirs)
```

Comment:
I'm not really sure why this is an option (on zapgen) rather than just assumed to be true.

Reply:
I did this based on some user awareness consideration. If a user makes a typo in the path and the command automatically creates and stores the zap file there and later the user doesn't find the file on the path what is in their mind, then it can cause unnecessary noise. 

However, I can change it if we want to.

Reply:
The trouble is that the user that did NOT make a mistake will get an error, the first time they run, as the directory won't exist.

Reply:
That is true, however it can be a warning for the user about there is a mistake/missing directory in their command or script. Moreover this method is a copy the underlying curl's behaviour. 

Again, I can change it if we want to.